manifest/product-config.json

```diff
@@ -2,6 +2,7 @@
     "product": "sync_gateway",
     "manifests": {
         "manifest/default.xml": {
+            "do-build": false,
```

Comment:
This looks like it's disabling the current release builds, which would be incorrect.

Reply:
right.  my bad.